d/dktest/dktest_ubi_8.3.sh

```diff
@@ -0,0 +1,62 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : dktest 
+# Version               : 0.3.0,0.3.6
+# Source repo           : https://github.com/dhui/dktest
+# Tested on             : UBI 8.3
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Arumugam N S<asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
+
+if [ -d "dktest" ] ; then
+  rm -rf dktest
+fi
+
+# Dependency installation
+sudo dnf install -y git golang 
+# Download the repos
+git clone https://github.com/dhui/dktest
+
+
+# Build and Test dktest 
+cd dktest 
+git checkout $VERSION
+ret=$?
+if [ $ret -eq 0 ] ; then
+ echo "$Version found to checkout "
+else
+ echo "$Version not found "
+ exit
+fi
+
+#Build and test
+go get -v -t ./...
+ret=$?
+if [ $ret -ne 0 ] ; then
+  echo "Build failed "
+else
+  #networktest is not supported
```

Comment:
Does it fail only in ppc64le ?

Reply:
Yes,Failed to pull image: nginx:alpine error,docker image nginx:alpine not supported at this ubi env.

Reply:
I see https://hub.docker.com/_/nginx supports ppc64le also. What command is failing here ?

Reply:
=== RUN   TestRunWithNetwork
    dktest.go:36: Pulling image: nginx:alpine
    dktest.go:184: Failed to pull image: nginx:alpine error: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--- FAIL: TestRunWithNetwork (0.00s)
=== RUN   TestRunWithNetworkPortBinding
    dktest.go:36: Pulling image: nginx:alpine
    dktest.go:184: Failed to pull image: nginx:alpine error: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
--- FAIL: TestRunWithNetworkPortBinding (0.00s)

not able to starting unix:///var/run/docker.sock from UBI

systemctl start docker

[root@673e43021f6c ~]# systemctl start docker
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down

tried to pull image 

[root@673e43021f6c ~]# docker pull nginx
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
cannot clone: Operation not permitted
Error: cannot re-exec process
[root@673e43021f6c ~]# podman start docker
cannot clone: Operation not permitted
Error: cannot re-exec process
[root@673e43021f6c ~]# podman start nginx
cannot clone: Operation not permitted
Error: cannot re-exec process
[root@673e43021f6c ~]#

[root@673e43021f6c ~]# docker run --name tmp-nginx-container -d nginx
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
cannot clone: Operation not permitted
Error: cannot re-exec process
[root@673e43021f6c ~]#

but it does not work ,pls share if you have any clue how to pull the image nginx and make daemon running.